File: college/lista-aula-3/js/questao-1.js
// 1) Leia 20 valores reais e calcule seu somatório utilizando a instrução while
const prompt = require('prompt-sync')({ sigint: true });

let soma = 0;
let index = 1;

while (index <= 20) {
  const entrada = Number(prompt(`Digite a entrada ${index}: `));

  if (!isNaN(entrada)) {
    soma += entrada;
    index += 1;
  } else {
    console.log('Digite apenas números.');
  }
}

console.log(`A soma é: ${soma}`);

File: python/lista-3-facul/questao-1.py
soma = 0;
index = 1;

while index <= 20:
  entrada = input(f'Digite a entrada {index}: ');

  if entrada.isnumeric():
    soma += int(entrada);
    index += 1;
  else:
    print('Digite apenas números.');

print(f'\nA soma é: {soma}');
